Code/Framework/AzFramework/AzFramework/Physics/Configuration/RigidBodyConfiguration.cpp

```diff
@@ -184,6 +203,25 @@ namespace AzPhysics
         return m_ccdEnabled;
     }
 
+    bool RigidBodyConfiguration::CcdReadOnly() const
+    {
+        if (auto* physicsSystem = AZ::Interface<AzPhysics::SystemInterface>::Get())
+        {
+            return !physicsSystem->GetDefaultSceneConfiguration().m_enableCcd || m_kinematic;
+        }
+        return m_kinematic;
+    }
+
+    AZStd::string_view RigidBodyConfiguration::GetCcdTooltip() const
+    {
+        return (m_kinematic) ? ccdDescriptionReadOnly : ccdDescription;
```

Comment:
nitpick - not a big deal but I think the brackets are redundant here and on line 222

Reply:
I usually add them as well cause they help me with readability, but yes they are not needed.